osu.Game/Overlays/Rankings/SpotlightSelector.cs

```diff
@@ -162,11 +164,33 @@ private class SpotlightsDropdown : OsuDropdown<APISpotlight>
 
             protected override DropdownMenu CreateMenu() => menu = base.CreateMenu().With(m => m.MaxHeight = 400);
 
+            protected override DropdownHeader CreateHeader() => new SpotlightsDropdownHeader();
+
             [BackgroundDependencyLoader]
             private void load(OverlayColourProvider colourProvider)
             {
+                // osu-web adds a 0.6 opacity container on top of the 0.5 base one when hovering, 0.8 on a single container here matches the resulting colour
+                AccentColour = colourProvider.Background6.Opacity(0.8f);
                 menu.BackgroundColour = colourProvider.Background5;
-                AccentColour = colourProvider.Background6;
+                Padding = new MarginPadding { Vertical = 20 };
+            }
+
+            private class SpotlightsDropdownHeader : OsuDropdownHeader
+            {
+                public SpotlightsDropdownHeader()
+                {
+                    Height = 48;
```

Comment:
This probably shouldn't be explicit too as it's computed on web. This means it'll look slightly wrong in the short run but will hopefully go back to being correct when font sizes are corrected for.

Reply:
Removed this and temporarily added padding to the text sprite to compensate for the size differences (like in other adjustment PRs).